6.3.0/aspnet-core/src/BhResturant.Core/Authorization/BhResturantAuthorizationProvider.cs

```diff
@@ -12,6 +12,8 @@ public override void SetPermissions(IPermissionDefinitionContext context)
             context.CreatePermission(PermissionNames.Pages_Users_Activation, L("UsersActivation"));
             context.CreatePermission(PermissionNames.Pages_Roles, L("Roles"));
             context.CreatePermission(PermissionNames.Pages_Tenants, L("Tenants"), multiTenancySides: MultiTenancySides.Host);
+            context.CreatePermission(PermissionNames.Pages_Products, L("Products"), multiTenancySides: MultiTenancySides.Host | MultiTenancySides.Tenant);
```

Comment:
here you had permission which user can get and set using the IProductApplicationService?

Reply:
creat permission and then you can register for it. if we want to defined a unique permission is  for each operation that needs to be authorized [](https://aspnetboilerplate.com/Pages/Documents/Authorization)